embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/api/delegate/OTelApiDelegate.kt

```diff
@@ -38,18 +36,4 @@ internal class OTelApiDelegate(
         }
         bootstrapper.openTelemetryModule.openTelemetryConfiguration.addLogExporter(logRecordExporter)
     }
-
-    override fun getTracer(instrumentationModuleName: String?, instrumentationModuleVersion: String?): Tracer {
-        return if (sdkCallChecker.started.get()) {
-            if (instrumentationModuleName == null) {
-                bootstrapper.openTelemetryModule.sdkTracer
```

Comment:
This was probably wrong anyway - to expose the exact SDK tracer instance we use. Now we close this loophole and always provide our wrapped implementation.